### PR TITLE
svm from scratch 

### PR DESCRIPTION
**Pull request by shubhamgit1**

### CODE REVIEW

File: assignment_3/chandel svm/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+4. Converted the training dataset into dictionary in such a way that with key as -1, values are negative class samples and with key as +1, values are positive class samples.
+
+5. Predicted the classes of test data and visualized the svm,by using the methods predict and visualize respectively.And visualization results were like:
+          ![image info](./svm_visualize.png)
+
+6. Compared with predicted results of svm from sklearn.
+
+
+##More into the Support_Vector_Machine class
```
Comment (AnirudhDagar) [inline assignment_3/chandel svm/README.md]: add space here.
## More ....

File: assignment_3/chandel svm/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      - visualize
+
+2. Generated a binary class toy dataset having 100 data points using make_blobs from sklearn.
+
+3. Splitted the dataset into training and testing data.
+
+4. Converted the training dataset into dictionary in such a way that with key as -1, values are negative class samples and with key as +1, values are positive class samples.
+
+5. Predicted the classes of test data and visualized the svm,by using the methods predict and visualize respectively.And visualization results were like:
+          ![image info](./svm_visualize.png)
```
Comment (AnirudhDagar) [inline assignment_3/chandel svm/README.md]:
```suggestion

          ![image info](./svm_visualize.png)
```

**Review by AnirudhDagar — CHANGES_REQUESTED** [review]
make these changes!

**Comment by shubhamgit1** [comment]
sir i made changes as you said

File: assignment_3/README.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-## Problem Statement
+11## Problem Statement
```
Comment (AnirudhDagar) [inline assignment_3/README.md]: remove this error!

**Review by AnirudhDagar — CHANGES_REQUESTED** [review]
Just a minor  error in 1st line. Chnage that. I'll merge.

**Comment by shubhamgit1** [comment]
sir please check i have removed that error

**Pull request merged**